addons/rose/tests/dummy/app/templates/index.hbs

```diff
@@ -1,196 +1,50 @@
-<h2>Condensed Table</h2>
```

Comment:
Ignore all the changes in this file, this is just the dummy test for development purposes.